recipe/0064-opera-one-canvas/manifest.json

```diff
@@ -0,0 +1,122 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "{{ id.url }}",
+  "type": "Manifest",
+  "label": { "it": [ "L'elisir d'amore" ], "en": [ "The elixir of love" ] },
+  "metadata": [
+    {
+      "label": { "en": [ "Date Issued" ] },
+      "value": { "en": [ "2019" ] }
+    },
+    {
+      "label": { "en": [ "Publisher" ] },
+      "value": { "en": [ "Indiana University Jacobs School of Music" ] }
+    }
+  ],
+  "items": [
+    {
+      "id": "{{ id.path }}/canvas/1",
+      "type": "Canvas",
+      "width": 1920,
+      "height": 1080,
+      "duration": 7278.422,
+      "items": [
+        {
+          "id": "{{ id.path }}/canvas/1/annotation_page/1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "{{ id.path }}/canvas/1/annotation_page/1/annotation/1",
+              "type": "Annotation",
+              "motivation": "painting",
+              "target": "{{ id.path }}/canvas/1",
```

Comment:
This may not be strictly necessary but as your targeting a specific time in the second canvas I think it would be more explicit if you add `#t=0` on the end of this target.